tests/apps/scout/test_scout_load_config.py

```diff
@@ -28,6 +29,28 @@ def test_validate_mt_bam(sample_dict):
     assert ind_obj.mt_bam == sample["mt_bam"]
 
 
+def test_validate_reviewer_alignment(sample_dict):
+    """Test to validate an individual"""
+    # GIVEN some sample information
+    sample = sample_dict
+    # WHEN validating the sample data
+    ind_obj = scout_load_config.ScoutMipIndividual(**sample)
+
+    # THEN assert that the reviewer alignment path is correct
```

Comment:
```suggestion
    # THEN assert that the reviewer alignment path is set
```
Correct does not really add much info 😄